packages/frontend/src/providers/Explorer/ExplorerProvider.tsx

```diff
@@ -1569,9 +1591,17 @@ const ExplorerProvider: FC<
         if (unsavedChartVersion.metricQuery.sorts.length <= 0 && defaultSort) {
             setSortFields([defaultSort]);
         } else {
-            return mutateAsync();
+            // force new results even when query is the same
+            clearQueryResults();
```

Comment:
Since we changed from `useMutation` to `useQuery` it only fetches if the arguments changed. 
To force new results we need to clear the react-query cache.